multi-node/aws/pkg/config/config.go

```diff
@@ -160,8 +164,22 @@ func (c Cluster) Config() (*Config, error) {
 	config.APIServers = fmt.Sprintf("http://%s:8080", c.ControllerIP)
 	config.SecureAPIServers = fmt.Sprintf("https://%s:443", c.ControllerIP)
 	config.APIServerEndpoint = fmt.Sprintf("https://%s", c.ExternalDNSName)
-	if config.UseCalico {
-		config.K8sNetworkPlugin = "cni"
+	config.K8sNetworkPlugin = "cni"
```

Comment:
See commit: https://github.com/coreos/coreos-kubernetes/pull/551/commits/bc6814694a0ced6b807cad56a06da99993038c37

We can setup both networks through CNI now, so everything uses CNI now.